tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932175.yaml

```
@@ -163,3 +163,107 @@ tests:
             version: HTTP/1.1
           output:
             log_contains: id "932175"
+  - test_title: 932175-10
+    desc: "Test for alias with -p argument"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+              Host: localhost
+              User-Agent: "OWASP CRS test agent"
+            method: POST
+            port: 80
+            uri: "/post"
+            data: |
+              rce=alias -p jzfrx=id
+            version: HTTP/1.1
+          output:
+            log_contains: id "932175"
+  - test_title: 932175-11
+    desc: "Test for alias with zsh flags"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+              Host: localhost
+              User-Agent: "OWASP CRS test agent"
+            method: POST
+            port: 80
+            uri: "/post"
+            data: |
+              rce=alias -gmr a=b
+            version: HTTP/1.1
+          output:
+            log_contains: id "932175"
+  - test_title: 932175-12
+    desc: "Test for alias with zsh '+' flags"
```

For all the tests that use an URL-encoded payload, please spell out the payload in clear text as part of `desc:`:
```suggestion
    desc: |
      Test for alias with zsh '+' flags.
      Payload: ...
```